ui/desktop/electron-app/src/services/client-daemon-manager.js

```diff
@@ -24,7 +24,7 @@ class ClientDaemonManager {
     const data = spawnSync(daemonStatusCommand);
 
     const status = jsonify(data);
-    this.#socketPath = status?.socket_address;
+    this.#socketPath = status?.item?.socket_address;
```

Comment:
? for learning: what is this change for?

Reply:
Answer for learning: The BE team updated this API response to nest `socket_address` inside of `item` to match their other API responses.